examples/notebooks/brats_segmentation_3d.ipynb

```diff
@@ -77,11 +77,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {
     "tags": []
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "MONAI version: 0.1.0+318.ga2cd842\n",
```

Comment:
it is still showing monai v0.1.0, could you run `git fetch --tags` and rerun this cell? might be a bug for the versioning

Reply:
Hi @wyli ,

I tried to `git fetch --tags`, it's still MONAI v0.1+.
I think maybe this is some bug in versioning tool, could you please help file a ticket for it?

Thanks.

Reply:
couldn't replicate this issue from my side

Reply:
What did you use to test? repo or pip wheel? I used `pip install monai` directly.
Thanks.

Reply:
I opened a new docker container, issue resolved.
Seems some cache in my previous docker container caused this issue.
Thanks.

Reply:
Hi @wyli , I already updated all the notebooks with latest MONAI and PyTorch version.
Thanks for raising this issue.

Reply:
thanks, I think we need to clarify the steps to run the notebook examples (as a part of https://github.com/Project-MONAI/MONAI/issues/872). probably we don't need `pip install monai` after git cloning the source code